src/components/standalone-link/standalone-link.module.css

```diff
@@ -35,22 +35,22 @@
 }
 
 .color-primary {
-  color: var(--token-color-action-foreground-on-faint);
+  color: var(--token-color-foreground-action);
 
   &:active {
-    color: var(--token-color-action-foreground-high-contrast);
+    color: var(--token-color-foreground-action-active);
   }
 
   &:hover {
-    color: var(--token-color-action-foreground-on-faint);
+    color: var(--token-color-foreground-action-hover);
```

Comment:
Was this component updated to have `color` change on `hover`? The values were previously the same.

Reply:
Yes, this update was merged in on Feb 3rd with [#155](https://github.com/hashicorp/design-system-components/pull/155).

Reply:
Same here: what @heatherlarsen said.